mlibrary/design-system-docs
# DS-149 Update packages for Feb 2023

## What's new

February package updates

Updated:
- `sass`
- `jsdom`

### To test

- run `npm install`
- Go through site
- Edit a `.scss` file and confirm build
